docs/development/making_pr.rst

```diff
@@ -0,0 +1,92 @@
+Making a PR to SecureDrop
+=============================
+
+Forking and Cloning the Project
+---------------------------------
+
+1. Fork SecureDrop on GitHub from the Main Repository to your own profile.
+
+2. Clone the forked repository.
+
+.. code:: sh
+
+  git clone https://github.com/<your-username>/securedrop.git
+  cd securedrop
+
+3. Add the Main Repository as an upstream remote.
+
+.. code:: sh
+
+  git remote add upstream https://github.com/freedomofpress/securedrop.git
+
+
+Make Your Changes and Push to the Fork
+----------------------------------------------
+
+Create a branch
+~~~~~~~~~~~~~~~~~~
+
+Create a branch on which you make your changes.
+
+.. code:: sh
+
+  git checkout -B change-one
+
+
+Make Your Changes and Commit
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Now enter the directory of your fork amd make changes as you wish.
+Run tests for the changes you have made.
+
+If you create a new file, remember to add it with :code:`git add`.
+
+.. code:: sh
+
+  git add <new-file>
+
+Commit your changes, adding a description of what was added. If you’re not
+used to Git, the simplest way is to commit all modified files and add a
+description message of your changes in a single command like this:
+
+.. code:: sh
+
+  git commit -a -m "<Description of changes made>"
+
+Merge Your Changes to develop branch
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The :code:`develop` branch is the main branch in the SecureDrop project.
+Merge all your changes into the :code:`develop` branch and delete the branch
+you made.
+
+.. code:: sh
+
+  git checkout develop
+  git merge change-one
+
+Pushing the changes to GitHub Fork
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+First get any updates made in the upstream repository.
+
+.. code:: sh
+
+  git pull upstream develop
+
+Once your changes are committed, push the changes to your GitHub fork.
+
+.. code:: sh
+
+  git push origin develop
+
+Making a Pull Request
+-----------------------
+
+1. Through GitHub make a pull request.
+
+2. Once PR is made, the Travis CI build server checks all tests and Codecov
+runs a report on test coverage. The reports are available in the
+PR page and also emailed to admins.
+
+3. From there, an admin accepts or rejects the PR.
```

Comment:
Here we can let the contributor know there may be a back and forth between the PR submitter and maintainer(s). Specifically we can say something like:

"From there, a maintainer will accept your PR or they may request comments for you to address prior to merge."

Reply:
One other area that new contributors often have trouble with is squashing commits - we can include a link to an external explainer for that:

"The maintainer may also ask you to [squash your commits](https://robots.thoughtbot.com/git-interactive-rebase-squash-amend-rewriting-history) prior to merge."